src/Spectre.libGenetic/Generation.cpp

```diff
@@ -44,6 +45,19 @@ Generation::Generation(std::vector<Individual> &&generation):
     else { }
 }
 
+
+Generation::Generation(size_t size, size_t individualSize, size_t initialFillup, Seed seed)
```

Comment:
I've written above about this functionality.

Reply:
done